code/game/objects/items/weapons/material/chainsaw.dm

```diff
@@ -0,0 +1,92 @@
+obj/item/weapon/material/twohanded/chainsaw
+	base_name = "Chainsaw"
+	desc = "Vroom vroom."
+	base_icon = "chainsaw"
+	var/on = 0
+	var/max_fuel = 30
+	w_class = ITEMSIZE_LARGE
+	slot_flags = SLOT_BACK
+	unwielded_force_divisor = 0.25
+	force_divisor = 0.5
+	dulled_divisor = 0.6
+	sharp = 1
+	edge = 1
+	w_class = ITEMSIZE_LARGE
+	slot_flags = SLOT_BACK
+	force_wielded = 35
```

Comment:
Is it intended to be so high up here?

Reply:
Force_wielded?

Reply:
Shouldn't force only be Super High when it's on, or am I misinterpreting this?

Reply:
True that. Will handle.